k8s/overlays/nerc-ocp-staging/patches/deployment-xdmod.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: xdmod-ui
+  labels:
+    component: xdmod-ui
+spec:
+  selector:
+    matchLabels:
+      component: xdmod-ui
+  template:
+    metadata:
+      labels:
+        component: xdmod-ui
+    spec:
+      containers:
+        - image: image-registry.openshift-image-registry.svc:5000/xdmod-staging/moc-xdmod:latest
+          name: xdmod
+      initContainers:
+        - image: image-registry.openshift-image-registry.svc:5000/xdmod-staging/moc-xdmod-dev:latest
```

Comment:
Just confirming that this container is intentionally using a different image than the others?

Reply:
Yes.  The initial init container needed some additional permissions to change permissions on some of the directories/files to get around permission issues.